examples/PET_simulation/ScatterEstimation.par

```diff
@@ -0,0 +1,95 @@
+Scatter Estimation Parameters :=
+
+;Run in debug mode
+;; A new folder called extras will be created, in which many 
+;; extra files will be stored 
+run in debug mode := 1
+
+;; 
+;; initialisation 
+;; 
+
+; If name is provided and recomputation == true
+; the file will be saved in the disk 
+
+recompute initial activity image := 1
+initial activity image filename := 
+
+; Smoothing for activity and attenuation images 
+zoom xy := 
+zoom z := 
+
+; Measured data
+input file := 
+
+; Multuplicative data 
+
+; Attenuation data
+recompute attenuation projdata := 
+attenuation projdata filename := 
+attenuation image filename := 
+
+; Normalisation coefficients
+normalisation coefficients filename :=
+
+; Mask 
+; If !recompute mask projdata then the filename must be set.
+recompute mask projdata := 
+mask projdata filename := 
+
+; Input or output filename - depends on recompute  
+;; If recompute is set, fwd will be performed automatically
+recompute mask image := 
+mask image filename := 
+
+mask max threshold := 
+mask add scaler := 
+mask min threshold := 
+mask times scalar :=
+mask prostfilter filename := 
+; End of Mask
+
+;Parameter file for the tail fitting of the mask image.
+tail fitting par filename := 
+
+;Backgroud data
+background projdata filename := 
+; export SSRB sinograms
+; export 2d projdata := 
+
+; ScatterSimulation Stuff 
+scatter simulation parameter file :=
+
+; Over-ride the values set in the scatter simulation parameteres file
```

Comment:
English is "override" (will need adjustment of keywords) Also typo "parameteres"

what are the values here? Booleans? These should all default to 1 then. Do we actually need this? (Wouldn't it be very strange not to override them?)

Reply:
I think it's confusing to have them. The ScatterEstimation should work with any scatter simulation,i.e. it shouldn't need to know what it's doing and therefore not know about its parameters. In addition, if we duplicate code, we'll trip ourselves up at some point.

Of course, someone might want to simulate scatter in a certain way, and then use ScatterEstimation with a different scatter estimator, but that is also true for image recon etc.

Let's remove them.